### ISSUE TITLE
feature:send to discord webhook url as C2 instead of on vps/local machine tcp tunnel

### ISSUE THREAD

**Issue by riggzwheelz** [Pirate-Devs/Kematian#6]
### What Feature?

Chainski made a version like that but it was deleted and I cannot find it. [http://web.archive.org/web/20240612232836/https://github.com/Chainski/Kematian-Stealer](url)
if someone can find this version for me or help me with that would be amazing. otherwise love the stealer. also beware of dual hooks from other stealers i heard of people doing that!

### Check the following before submitting (If Applicable)

- [X] I have checked the existing [issues](https://github.com/KematianPirate-Devs/Kematian/issues)
- [X] I want to work on this feature

**Comment by KDot227** [comment]
> ### What Feature?
> 
> Chainski made a version like that but it was deleted and I cannot find it. [http://web.archive.org/web/20240612232836/https://github.com/Chainski/Kematian-Stealer](url) if someone can find this version for me or help me with that would be amazing. otherwise love the stealer. also beware of dual hooks from other stealers i heard of people doing that!
> ### Add Screenshots (If Applicable)
> 
> _No response_
> ### Check the following before submitting (If Applicable)
> 
>     * [x]  I have checked the existing [issues](https://github.com/KematianPirate-Devs/Kematian/issues)
> 
>     * [x]  I want to work on this feature

discord auto bans now so we got rid of it.

**Comment by riggzwheelz** [comment]
Yes I understand that but would still like access to that version if possible?
